Automattic/wp-calypso
# Jetpack Connect: Fix ESLint warnings in examples

This PR fixes all of the ESLint warnings within the example components - previously we were using the `jetpack-connect__` prefix, but this PR proposes that we use `example-components__` as per the coding standards. There were also some large line length warnings.

The PR also fixes a couple of minor warnings that were present in the `sso` and `help-button` subcomponents. 

To verify there are no more ESLint warnings, you can run the following 2 commands in your calypso root dir:

* `./node_modules/.bin/eslint --cache client/signup/jetpack-connect/*.jsx`
* `./node_modules/.bin/eslint --cache client/signup/jetpack-connect/example-components/*.jsx`

If there are no warnings, no output should be displayed by the command.

To test:

* Checkout this branch
* Go to `/jetpack/connect/`
* Try connecting a site without Jetpack installed
* Try connecting a site with Jetpack >= 4.2 installed, but not activated
* Try connecting a site with Jetpack < 4.2 installed, but not activated
* Verify there are no regressions in any of the instructions pages
* Verify there are no ESLint warnings left

/cc @johnHackworth @roccotripaldi 

Test live: https://calypso.live/?branch=update/jetpack-connect-eslint-fixes
